test/e2e/simple_test.go

```diff
@@ -69,7 +69,10 @@ func TestE2ESimple(t *testing.T) {
 
 	totalTxs := 0
 	for _, block := range blockchain {
-		require.Equal(t, v1.Version, block.Version.App)
+		// the app version in the genesis is set to the latest version,
+		// so we should see this in the blocks
+		require.Equal(t, appconsts.LatestVersion,
+			block.Version.App)
```

Comment:
indentation still looks off to me. This assertion is easier to read on one line

```suggestion
		require.Equal(t, appconsts.LatestVersion, block.Version.App)
```